libs/@blockprotocol/core/src/types.ts

```diff
@@ -237,3 +237,4 @@ export type HtmlBlockDefinition = {
    */
   source?: string;
 };
+export type CoreHandlerCallback = (handler: CoreHandler) => void;
```

Comment:
```suggestion

export type CoreHandlerCallback = (handler: CoreHandler) => void;
```

Reply:
WebStorm has started messing with spacing between lines 😅 thank you!